sirepo/job_supervisor.py

```diff
@@ -574,7 +573,8 @@ def set_situation(self, op, situation, exception=None):
     @classmethod
     def __db_file(cls, computeJid):
         return _DB_DIR.join(
-            computeJid + sirepo.const.JSON_SUFFIX,
+            sirepo.simulation_db.assert_sim_db_file_path(computeJid)
```

Comment:
I'm confused by this. I don't believe __db_file (db/supervisor-job) has anything to do with sim_db_file (db/user/<uid>/<sim_type>/<sid>) so I'm not sure why that method would be called here. Maybe it needs to be renamed to capture multiple uses?

Reply:
This is poor naming of assert_sim_db_file_path, but it is the function I want to use. I don't want the computeJid to be something that's unsafe.

I'm going to ignore this for now. sim_db_file is a complicated name.